en_US.json

```diff
@@ -1317,7 +1317,7 @@
   "SKIP_CUTSCENE_DESC": "Skips the currently playing cutscene",
   "SLOT": "Slot",
   "SMART_KICK": "Smart Kick",
-  "SMART_KICK_DESC": "Kicks the player with the most appropriate method",
+  "SMART_KICK_DESC": "Kicks the player with the most appropriate method. \nIf you're host: Breakup \nIf target is host: OOM \nElse: Desync",
```

Comment:
> I think this is overkill

There's no way to know what the method does unless you know where to find the multi-kick code and how to understand it, i think it's fine to have an explanation, not anyone will go straight to github and look for the specific lines of code, when i first saw this button i kinda hesitated to use it because i didn't know how it worked... it might look like overkill but it's necessary